generated/components.json

```diff
@@ -4370,7 +4370,7 @@
           "name": "resize",
           "type": "'both' | 'horizontal' | 'vertical' | 'none'",
           "defaultValue": "'both'",
-          "description": "Changes background color to a higher contrast color"
+          "description": "Changes the resize behavior"
```

Comment:
Oo generating components.json doesn't run on forked PRs? 🤔 Because this was fixed in this [PR](https://github.com/primer/react/pull/3467)

Reply:
Oh I think it is not to do with the forked repo PRs, I guess it is related to https://github.com/primer/react/pull/3479?

Reply:
@broccolinisoup I think you're right that the generate command will fail on PRs from forks since they don't have access to the correct secrets in order to push back to the origin branch 😞 (since it runs in the CI job)